docker/docker-compose.yaml

```diff
@@ -0,0 +1,26 @@
+version: '2'
+services:
+  mongodb:
+    image: mongo:4.4.15
```

Comment:
I think there should be a comment or option here: further to my discussion on Discord, I noticed that you need to use version 4 if you want to run mongo on Pi. Else, you can use the latest version.